### PR TITLE
Test reshard feature after bucket owner is changed

### PR DESCRIPTION
**Pull request by TejasC88**
Polarion ID : CEPH-83574627

Change bucket owner and verify reshard operation happens as expected on the bucket.
Pass log : http://magna002.ceph.redhat.com/cephci-jenkins/tchandra/chown_reshard.txt

### CODE REVIEW

File: rgw/v2/tests/s3_swift/multisite_configs/test_bucket_chown_reshard.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+config:
```
Comment (anrao19) [inline rgw/v2/tests/s3_swift/multisite_configs/test_bucket_chown_reshard.yaml:1]: Can we add polarion-id and script details in the config file?

File: rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py [hunk 2]
```diff
@@ -160,6 +160,20 @@ def test_exec(config, ssh_con):
             )
```
Comment (anrao19) [inline rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py:160]: Can we add config details in the documentation section..

**Review by viduship — APPROVED** [review]

File: rgw/v2/tests/s3_swift/multisite_configs/test_bucket_chown_reshard.yaml (hunk 1 above)
Comment (TejasC88) [reply rgw/v2/tests/s3_swift/multisite_configs/test_bucket_chown_reshard.yaml:3]: Done

File: rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py (hunk 2 above)
Comment (TejasC88) [reply rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py:161]: Done

**Review by anrao19 — APPROVED** [review]
LGTM

**Review by ckulal — APPROVED** [review]

**Pull request merged**